widgets/heatmap/internal/axes/label.go

```diff
@@ -36,21 +38,85 @@ type Label struct {
 // the labels will be placed and aligned.
 // Labels are returned with Y coordinates in ascending order.
 // Y coordinates grow down.
-func yLabels(graphHeight, labelWidth int, labels []string) ([]*Label, error) {
-	return nil, errors.New("not implemented")
+func yLabels(labelWidth int, labels []string) ([]*Label, error) {
+	if min := 0; labelWidth < min {
+		return nil, fmt.Errorf("cannot place labels on a canvas with width %d, minimum is %d", labelWidth, min)
+	}
+
+	var ret []*Label
+	for row, l := range labels {
+		label, err := rowLabel(row, l, labelWidth)
+		if err != nil {
+			return nil, err
+		}
+		ret = append(ret, label)
+	}
+
+	return ret, nil
 }
 
-// rowLabel returns one label for the specified row.
+// rowLabel returns one Y label for the specified row.
 // The row is the Y coordinate of the row, Y coordinates grow down.
 func rowLabel(row int, label string, labelWidth int) (*Label, error) {
-	return nil, errors.New("not implemented")
+	// The area available for the Y label
+	ar := image.Rect(0, row, labelWidth, row+1)
+
+	pos, err := alignfor.Text(ar, label, align.HorizontalRight, align.VerticalMiddle)
+	if err != nil {
+		return nil, fmt.Errorf("unable to align the label value: %v", err)
```

Comment:
We have two error messages here that both say the exact same thing:

`unable to align the label value`

Can we add information about which label they are talking about to help disambiguate between them? E.g:

For here: `unable to align the Y label value: %v`
For the one below: `unable to align the X label value: %v`